accommodation-search-form/accommodation-search-form.js

```diff
@@ -0,0 +1,247 @@
+    showPromocode: true,
+
+    monthsShort: [
+        "Jan",
+        "Feb",
+        "Mar",
+        "Apr",
+        "May",
+        "Jun",
+        "Jul",
+        "Aug",
+        "Sep",
+        "Oct",
+        "Nov",
+        "Dec",
+    ],
+};
+
+const renderHotels = (hotels) => {
+    let optEle = "";
+    hotels.forEach((hotel) => {
+        optEle += `<option value="${hotel}">${hotel}</option>`;
+    });
+    return `
+    <div class="grid px-3 pt-4 pb-7">
+        <label class="form_label" for="hotel-selector">Location</label>
+        <select id="hotel-selector" name="hotel-selector">
+            <option value="" data-be="" selected>Select Hotel</option>
+            ${optEle}
+        </select>
+    </div>`;
+};
+const renderDate = (type, options) => {
+    let dateString = "",
+        dateValue = "";
+    if (type === "check-in") {
+        dateValue =
+            options.checkIn.getDate() +
+            ` ${options.monthsShort[options.checkIn.getMonth()]}`;
+        dateString =
+            options.checkIn.getDate() +
+            ` <small>${
+                options.monthsShort[options.checkIn.getMonth()]
+            }</small>`;
+    } else if (type === "check-out") {
+        dateValue =
+            options.checkOut.getDate() +
+            ` ${options.monthsShort[options.checkOut.getMonth()]}`;
+        dateString =
+            options.checkOut.getDate() +
+            ` <small>${
+                options.monthsShort[options.checkOut.getMonth()]
+            }</small>`;
+    }
+
+    let valEle = "";
+    if (options.showDatesLabel) {
+        valEle = `<input type="text" class="hidden" name="${type}" id="datepicker-${type}" value="${dateValue}"/>
+                <label class="datepicker text-center" for="datepicker-${type}" id="datepicker-${type}-label">${dateString}</label>`;
+    } else {
+        valEle = `<input type="text" class="" name="${type}" id="datepicker-${type}" value="${dateValue}"/>`;
+    }
+    return `
+    <div class="grid px-3 pt-4 pb-7">
+        <label class="form_label text-center">${
+            type === "check-in"
+                ? "Check In"
+                : type === "check-out"
+                ? "Check Out"
+                : ""
+        }</label>
+        ${valEle}
+    </div>`;
+};
+const renderRooms = () => {
+    return `
+    <div class="grid px-3 pt-4 pb-7">
+        <label class="form_label" for="rooms-selector">Rooms</label>
+        <select id="rooms-selector" name="rooms-selector">
+            <option selected="" value="1">1</option>
+            <option value="2">2</option>
+            <option value="3">3</option>
+            <option value="4">4</option>
+            <option value="5">5</option>
+            <option value="6">6</option>
+            <option value="7">7</option>
+            <option value="8">8</option>
+            <option value="9">9</option>
+            <option value="10">10</option>
+        </select>
+    </div>`;
+};
+const renderGuest = () => {
```

Comment:
I feel like this is something should be configureable, for example someone might allow more than 10 guests, some might have limitations of 5 at once.